aion/web/elm/src/View.elm

```diff
@@ -16,10 +18,41 @@ view model =
         [ page model ]
 
 
-layout : Html msg -> Html msg
-layout content =
-    div [ style [ ( "font-family", "'Roboto', sans-serif" ) ] ]
-        [ content ]
+layout : Html Msg -> Model -> Html Msg
+layout content model =
```

Comment:
You could pass only Navbar state instead of model, I think it makes it much easier because you instantly get the idea what part of state is going to be used in this function. Here: only navbar -> you're probably dealing with navbar.